### PR TITLE
Document new jpackcore option '-x'

### PR DESCRIPTION
**Pull request by keithc-ca**

### CODE REVIEW

**Comment by keithc-ca** [comment]
Rebased to address merge conflict.

**Comment by doveye** [comment]
Jenkins doc stage

File: docs/version0.27.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The following new features and notable changes since v 0.26.0 are included in th
 - [Stop parsing the JAVA_OPTIONS environment variable](#stop-parsing-the-java_options-environment-variable)
 - [Global lock reservation enabled by default](#global-lock-reservation-enabled-by-default)
 - [Increase in default operating system stack size on PPC64 platforms](#increase-in-default-operating-system-stack-size-on-ppc64-platforms)
+- [New `-x` option added to `jpackcore` / `jextract`](##new-x-option-recognized-by-jpackcore-jextract)
```
Comment (doveye) [inline docs/version0.27.md:36]: The link is not working in the draft (https://eclipse-openj9.github.io/openj9-docs-staging/808/version0.27) - one of the # symbols needs removing.

File: docs/version0.27.md [hunk 2]
```diff
@@ -79,6 +80,10 @@ Global lock reservation is now enabled by default. This is an optimization targe
 
 The default operating system stack size on AIX and Linux PPC64 is increased from 256KB to 512KB. You can change the operating system stack size by using the [-Xmso](xmso.md) option.
 
+### New `-x` option recognized by `jpackcore` / `jextract`
+
+The new option, `-x`, causes the system dump to be omitted from the archive created. In its place, the file `excluded-files.txt` is added which names the excluded file.
+
 ## Full release information
```
Comment (doveye) [inline docs/version0.27.md:87]: Would be good to add a link to the jpackcore topic, eg "For more information about this tool, see [Dump extractor (jpackcore)](tool_jextract.md). Also, there should be a comma before "which".

**Review by doveye — CHANGES_REQUESTED** [review]

File: docs/version0.27.md (hunk 2 above)
Comment (keithc-ca) [reply docs/version0.27.md:87]: I added the comma and a similar sentence, copied from `diag_overview.md`.

File: docs/version0.27.md (hunk 1 above)
Comment (keithc-ca) [reply docs/version0.27.md]: Fixed.

**Comment by keithc-ca** [comment]
Jenkins doc stage

**Pull request merged**